### PR TITLE
Cleanup `new Color/Matrix()`

### PR DESCRIPTION
**Pull request by LeviPesin**
Related issue: https://github.com/mrdoob/three.js/pull/26021

**Description**

Some clean up.

### CODE REVIEW

**Comment by LeviPesin** [comment]
Seems like this breaks something... I will test it locally.

**Comment by Mugen87** [comment]
Please consider to update to too many things at once.

**Comment by LeviPesin** [comment]
OK, reverted ShaderNode changes.

File: examples/jsm/loaders/NRRDLoader.js [hunk 1]
```diff
@@ -445,34 +443,25 @@ class NRRDLoader extends Loader {
...
 
-
 		if ( ! headerObject.vectors ) {
 
-			volume.matrix.set(
-				1, 0, 0, 0,
-				0, 1, 0, 0,
-				0, 0, 1, 0,
-				0, 0, 0, 1 );
+			volume.matrix = volume.inverseMatrix = new Matrix4();
```
Comment (mrdoob) [inline examples/jsm/loaders/NRRDLoader.js:448]: This doesn't seem right?

Comment (Mugen87) [reply examples/jsm/loaders/NRRDLoader.js:448]: Both properties point to the same object now.

Comment (mrdoob) [reply examples/jsm/loaders/NRRDLoader.js:448]: Why do they have to point to the same matrix?

Comment (LeviPesin) [reply examples/jsm/loaders/NRRDLoader.js:448]: Just because it looks a little bit simpler in one line. But OK, I can make it two lines.

Comment (mrdoob) [reply examples/jsm/loaders/NRRDLoader.js]: @LeviPesin Do you mind confirming that you understand what @Mugen87 wrote above? 😬

File: examples/jsm/loaders/NRRDLoader.js [hunk 2]
```diff
@@ -419,9 +419,7 @@ class NRRDLoader extends Loader {
 		const spacingZ = new Vector3().fromArray( headerObject.vectors[ 2 ] ).length();
 		volume.spacing = [ spacingX, spacingY, spacingZ ];
 
-
 		// Create IJKtoRAS matrix
```
Comment (mrdoob) [inline examples/jsm/loaders/NRRDLoader.js:422]: Shouldn't this comment be moved too?

File: examples/jsm/loaders/NRRDLoader.js (hunk 1 above)
Comment (mrdoob) [reply examples/jsm/loaders/NRRDLoader.js]: I guess the thing to understand is that these kind of optimizations can easily create very confusing bugs.

Comment (LeviPesin) [reply examples/jsm/loaders/NRRDLoader.js]: OK. It is changed now.

File: examples/jsm/loaders/NRRDLoader.js (hunk 2 above)
Comment (LeviPesin) [reply examples/jsm/loaders/NRRDLoader.js:422]: I think no, because `transitionMatrix` is a part of building IJKtoRAS matrix.

**Comment by Mugen87** [comment]
There are too many uncertainties in this PR and I don't think the new code styles are better then the previous ones. In no way such type of "clean-up" PR should require such an in-depth review. I would say it's better to close the PR.

**Comment by LeviPesin** [comment]
There was only one "uncertainty" in this PR (assigning the same object to two fields). I don't see why it can't be merged?

**Comment by LeviPesin** [comment]
Can we merge this PR?

**Comment by Mugen87** [comment]
Same as https://github.com/mrdoob/three.js/pull/26317#issuecomment-1607154750.

**Pull request closed without merging**